e2e/tests/adminAddCaseFlag_test.js

```diff
@@ -9,7 +9,7 @@ const selectors = {
   nextStepDropdown: "#next-step",
 };
 
-Feature("Add Case Flag @fixed");
```

Comment:
I thought this is fixed and changes are merged into master yesterday?  May be put the @fixed back?